app/views/categories/index.html.erb

```diff
@@ -0,0 +1,40 @@
+
+<div class="row"> 
+<div class="col-sm-4"> </div>
+<div class="col-sm-4">
+
+  <h2> Categories </h2>
+
+  <% @categories.each do |category| %>
+
+     <div class="row category_name">
+        <div class="col-sm-12">
+        <%= category.name %>
+        </div>
+     </div>
+
+    <div class="display_course hidden">
+
+      <% category.courses.each do |course| %>
+
+          <div class="row" style="margin-left: 1vw;">
+            <div class="col-sm-12">
+            <%= link_to(course.name, "courses/" + course.id.to_s + "/show")%>
```

Comment:
Always try to use rails `path` or `url` helpers`.
https://guides.rubyonrails.org/routing.html#generating-paths-and-urls-from-code
If you add i.e. `as: 'course'`, you'll get `course_path(course)`.